### PR TITLE
[docs] Changed code examples and bumped dev dependency

### PR DESCRIPTION
**Pull request by javier**
Changed docs to use the trades dataset. 
Bumped the braces dependency to fix an audit problem

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,77 +22,80 @@ For more details, please, check the <a href="Sender.html">Sender</a>'s documenta
...
+  const sender = Sender.fromConfig("http::addr=127.0.0.1:9000")
+
+  // add rows to the buffer of the sender
+  await sender
+    .table("trades")
+    .symbol("symbol", "ETH-USD")
+    .symbol("side", "sell")
+    .floatColumn("price", 2615.54)
+    .floatColumn("amount", 0.00044)
+    .at(Date.now(), 'ms')
```
Comment (puzpuzpuz) [inline README.md:38]: nit: we should use `'` or `"` quotes consistently
```suggestion
    .at(Date.now(), "ms")
```

File: README.md [hunk 2]
```diff
@@ -22,77 +22,80 @@ For more details, please, check the <a href="Sender.html">Sender</a>'s documenta
...
-    .then(console.log)
-    .catch(console.error);
+run().then(console.log).catch(console.error)
 ```
 
 ### Authentication and secure connection
 
 ```javascript
-const { Sender } = require('@questdb/nodejs-client');
+const { Sender } = require('@questdb/nodejs-client')
```
Comment (puzpuzpuz) [inline README.md:55]:
```suggestion
const { Sender } = require("@questdb/nodejs-client")
```

File: README.md [hunk 3]
```diff
@@ -22,77 +22,80 @@ For more details, please, check the <a href="Sender.html">Sender</a>'s documenta
...
 
 async function run() {
     // create a sender using HTTPS protocol with username and password authentication
-    const sender = Sender.fromConfig('https::addr=localhost:9000;username=user1;password=pwd');
+    const sender = Sender.fromConfig('https::addr=127.0.0.1:9000;username=admin;password=quest;')
```
Comment (puzpuzpuz) [inline README.md:59]:
```suggestion
    const sender = Sender.fromConfig("https::addr=127.0.0.1:9000;username=admin;password=quest;")
```

File: README.md [hunk 4]
```diff
@@ -22,77 +22,80 @@ For more details, please, check the <a href="Sender.html">Sender</a>'s documenta
...
-        .floatColumn('bid', 1.0197).floatColumn('ask', 1.0224)
-        .at(Date.now(), 'ms');
-    await sender.flush();
+    await sender
+        .table("trades")
+        .symbol("symbol", "ETH-USD")
+        .symbol("side", "sell")
+        .floatColumn("price", 2615.54)
+        .floatColumn("amount", 0.00044)
+        .at(Date.now(), 'ms')
```
Comment (puzpuzpuz) [inline README.md:68]:
```suggestion
        .at(Date.now(), "ms")
```

File: README.md [hunk 5]
```diff
@@ -22,77 +22,80 @@ For more details, please, check the <a href="Sender.html">Sender</a>'s documenta
...
 
-run().catch(console.error);
+run().catch(console.error)
 ```
 
 ### TypeScript example
 
 ```typescript
-import { Sender } from '@questdb/nodejs-client';
+import { Sender } from '@questdb/nodejs-client'
```
Comment (puzpuzpuz) [inline README.md:81]:
```suggestion
import { Sender } from "@questdb/nodejs-client"
```

File: README.md [hunk 6]
```diff
@@ -22,77 +22,80 @@ For more details, please, check the <a href="Sender.html">Sender</a>'s documenta
...
 
-async function run(): Promise<number> {
+async function run(): Promise<void> {
     // create a sender using HTTPS protocol with bearer token authentication
-    const sender: Sender = Sender.fromConfig('https::addr=localhost:9000;token=Xyvd3er6GF87ysaHk');
+    const sender: Sender = Sender.fromConfig('https::addr=127.0.0.1:9000;token=Xyvd3er6GF87ysaHk;')
```
Comment (puzpuzpuz) [inline README.md:85]:
```suggestion
    const sender: Sender = Sender.fromConfig("https::addr=127.0.0.1:9000;token=Xyvd3er6GF87ysaHk;")
```

File: README.md [hunk 7]
```diff
@@ -22,77 +22,80 @@ For more details, please, check the <a href="Sender.html">Sender</a>'s documenta
...
-    sender.table('prices').symbol('instrument', 'EURUSD')
-        .floatColumn('bid', 1.0197).floatColumn('ask', 1.0224).at(Date.now(), 'ms');
-    await sender.flush();
+    await sender
+        .table("trades")
+        .symbol("symbol", "ETH-USD")
+        .symbol("side", "sell")
+        .floatColumn("price", 2615.54)
+        .floatColumn("amount", 0.00044)
+        .at(Date.now(), 'ms')
```
Comment (puzpuzpuz) [inline README.md:94]:
```suggestion
        .at(Date.now(), "ms")
```

File: README.md [hunk 8]
```diff
@@ -101,77 +104,79 @@ run().catch(console.error);
 ### Worker threads example
 
 ```javascript
-const { Sender } = require('@questdb/nodejs-client');
-const { Worker, isMainThread, parentPort, workerData } = require('worker_threads');
+const { Sender } = require('@questdb/nodejs-client')
```
Comment (puzpuzpuz) [inline README.md:107]:
```suggestion
const { Sender } = require("@questdb/nodejs-client")
```

File: README.md [hunk 9]
```diff
@@ -101,77 +104,79 @@ run().catch(console.error);
...
+const { Worker, isMainThread, parentPort, workerData } = require('worker_threads')
```
Comment (puzpuzpuz) [inline README.md:108]:
```suggestion
const { Worker, isMainThread, parentPort, workerData } = require("worker_threads")
```

File: README.md [hunk 10]
```diff
@@ -101,77 +104,79 @@ run().catch(console.error);
...
-// generates random prices for a ticker for max 5 seconds, then the feed closes
+// generates random prices and amounts for a ticker for max 5 seconds, then the feed closes
 function* venue(ticker) {
-    let end = false;
-    setTimeout(() => { end = true; }, rndInt(5000));
+    let end = false
+    setTimeout(() => { end = true; }, rndInt(5000))
     while (!end) {
-        yield {'ticker': ticker, 'price': Math.random()};
+        yield {'ticker': ticker, 'price': Math.random(), 'amount': Math.random()}
```
Comment (puzpuzpuz) [inline README.md:116]:
```suggestion
        yield {ticker, price: Math.random(), amount: Math.random()}
```

File: README.md [hunk 11]
```diff
@@ -101,77 +104,79 @@ run().catch(console.error);
...
-        await sleep(rndInt(30));
+        await onTick(tick)
+        await sleep(rndInt(30))
     }
 }
 
 async function run() {
     if (isMainThread) {
-        const tickers = ['t1', 't2', 't3', 't4'];
+        const tickers = ['ETH-USD', 'BTC-USD', 'SOL-USD', 'DOGE-USD']
```
Comment (puzpuzpuz) [inline README.md:133]:
```suggestion
        const tickers = ["ETH-USD", "BTC-USD", "SOL-USD", "DOGE-USD"]
```

File: README.md [hunk 12]
```diff
@@ -101,77 +104,79 @@ run().catch(console.error);
...
                 .on('message', (msg) => {
-                    console.log(`Ingested ${msg.count} prices for ticker ${msg.ticker}`);
+                    console.log(`Ingested ${msg.count} prices for ticker ${msg.ticker}`)
                 });
         }
     } else {
         // it is important that each worker has a dedicated sender object
         // threads cannot share the sender because they would write into the same buffer
-        const sender = Sender.fromConfig('http::addr=localhost:9000');
+        const sender = Sender.fromConfig('http::addr=127.0.0.1:9000');
```
Comment (puzpuzpuz) [inline README.md:146]:
```suggestion
        const sender = Sender.fromConfig("http::addr=127.0.0.1:9000");
```

File: README.md [hunk 13]
```diff
@@ -101,77 +104,79 @@ run().catch(console.error);
...
-                .table('prices')
-                .symbol('ticker', tick.ticker)
-                .floatColumn('price', tick.price)
-                .at(Date.now(), 'ms');
+                .table("trades")
+                .symbol("symbol", tick.ticker)
+                .symbol("side", "sell")
+                .floatColumn("price", tick.price)
+                .floatColumn("amount", tick.amount)
+                .at(Date.now(), 'ms')
```
Comment (puzpuzpuz) [inline README.md:158]:
```suggestion
                .at(Date.now(), "ms")
```

File: README.md [hunk 14]
```diff
@@ -101,77 +104,79 @@ run().catch(console.error);
...
             await sender.flush();
             count++;
         });
 
         // let the main thread know how many prices were ingested
-        parentPort.postMessage({'ticker': workerData.ticker, 'count': count});
+        parentPort.postMessage({'ticker': workerData.ticker, 'count': count})
```
Comment (puzpuzpuz) [inline README.md:164]:
```suggestion
        parentPort.postMessage({ticker: workerData.ticker, count})
```

File: examples/workers.js [hunk 15]
```diff
@@ -1,69 +1,75 @@
...
 function* venue(ticker) {
-  let end = false;
-  setTimeout(() => { end = true; }, rndInt(5000));
-  while (!end) {
-    yield {'ticker': ticker, 'price': Math.random()};
-  }
+    let end = false
+    setTimeout(() => { end = true; }, rndInt(5000))
+    while (!end) {
+        yield {'ticker': ticker, 'price': Math.random(), 'amount': Math.random()}
```
Comment (puzpuzpuz) [inline examples/workers.js:10]:
```suggestion
        yield {ticker, price: Math.random(), amount: Math.random()}
```

File: examples/workers.js [hunk 16]
```diff
@@ -1,69 +1,75 @@
...
+                .floatColumn("amount", tick.amount)
+                .at(Date.now(), 'ms')
+            await sender.flush()
+            count++;
+        });
 
-    // let the main thread know how many prices were ingested
-    parentPort.postMessage({'ticker': workerData.ticker, 'count': count});
+        // let the main thread know how many prices were ingested
+        parentPort.postMessage({'ticker': workerData.ticker, 'count': count})
```
Comment (puzpuzpuz) [inline examples/workers.js:58]:
```suggestion
        parentPort.postMessage({ticker: workerData.ticker, count})
```

File: README.md [hunk 17]
```diff
@@ -22,77 +22,80 @@ For more details, please, check the <a href="Sender.html">Sender</a>'s documenta
...
-        .at(Date.now(), 'ms');
-    await sender.table('prices').symbol('instrument', 'GBPUSD')
-        .floatColumn('bid', 1.2076).floatColumn('ask', 1.2082)
-        .at(Date.now(), 'ms');
-
-    // flush the buffer of the sender, sending the data to QuestDB
-    // the buffer is cleared after the data is sent, and the sender is ready to accept new data
-    await sender.flush();
-
-    // add rows to the buffer again, and send it to the server
```
Comment (glasstiger) [inline README.md:43]: This part of the example was supposed to demonstrate that you usually use the client by adding rows, then flush(), adding rows again, flush() again, and repeat until you call close() at the end.
Maybe it is not that important though, especially that auto-flushing is the default.

**Review by glasstiger — APPROVED** [review]

**Pull request merged**